modules/mobile-agents/android/client/plugins/IDPProxy/src/org/wso2/mdm/agent/proxy/utils/ServerUtilities.java

```diff
@@ -247,19 +326,19 @@ public static HttpClient getCertifiedHttpClient() throws IDPTokenManagerExceptio
 				KeyStore localTrustStore = KeyStore.getInstance("BKS");
 				inStream =
 						IdentityProxy.getInstance().getContext().getResources()
-						             .openRawResource(R.raw.emm_truststore);
+								.openRawResource(R.raw.emm_truststore);
 				localTrustStore.load(inStream, Constants.TRUSTSTORE_PASSWORD.toCharArray());
 
 				SchemeRegistry schemeRegistry = new SchemeRegistry();
 				schemeRegistry.register(new Scheme("http", PlainSocketFactory.getSocketFactory(),
-				                                   80));
+						80));
```

Comment:
What's 80? Let's avoid using 'magic numbers' within any implementation. Instead, read these values from a configuration file. Also use appropriate constants upon the same.